test/base/test_prior.py

```diff
@@ -125,7 +125,7 @@ def test_mode(scale, prior_type_list):
 
             # flat functions don't have local minima, so dont check this
             # for uniform priors
-
```

Comment:
Is this an actual output we want? especially since it is a print?

Reply:
Ooops, definitely not.